modules-management.md

```diff
@@ -0,0 +1,52 @@
+# Proposal: Proposals - Change how we manage modules in Ansible (the product) and the Ansible Community
+
+ - Date: 09 AUGUST, 2016
+
+ - Status: Rewritten from June proposal
+
+ - Proposal Type: Community Development Process.
+
+ - Targeted Release: 2.2 or possibly 2.3
+
+ - Estimated time to Implement: 3-4 months.
+
+## Motivation and Problems
+There are currently two module repositories: ansible-modules-core, and ansible-modules-extras.  Traditionally this has been recognized to mean that there is a standard for Core modules (since they are to be supported by the Ansible Core team and Red Hat) and Extras (modules that are part of Ansible, but more broadly contributed to and less supported).  In practice, Extras became roughly the same thing as Core since both Core and Extras are distributed with Ansible and supported by the Ansible Core team.  
+
+That condition had the unintended consequences of forcing the Core team to set the requirements for Extras modules to be roughly the same as Core.  The result was that lots of modules contributed by the Ansible community were either rejected or asked to reach a certain level of robustness or feature-completion for inclusion.  This gave the community little room to have "starter" functionality, or a place to contribute simpler or differentiated functionality that is not necessarily a part of the "Core" product, but is nonetheless a good place to get other functionality into ansible.
+
+Having multiple repos also made it more difficult for contributors to contribute things besides the module itself.  module_utils and tests for modules would have to be contributed to the main ansible repository even though the module itself lived in the ansible-modules-extras repository.  Bugs for modules might end up submitted against the main ansible repository or the ansible-modules-core repository even though the module lived in ansible-modules-extras.
+
+## Solution
+Ansible Core team and Contributors (from the Contributor Summit at the 2016 Ansible Fest San Francisco) propose to create categories of ansible Modules where Modules are grouped by a rough category of like-functionality (Nagios Modules would be grouped together, as would AWS, VMWare, etc).  There are a large number of implications for that working in the Ansible workflow and Community:
+  - There *will not* be multiple repos.  There will be one overall repo in github for all of Ansible.
```

Comment:
There will be one overall repo in Github for all of Ansible including all modules.